snippets/mapped-table.md

```diff
@@ -0,0 +1,35 @@
+### MappedTable
+
+Generate a table with dynamic rows based on passed array of data. Optionally include headers.
+
+Conditionaly check weather to include table headers or not. Use `Array.prototype.map` to map over array of data and return `<tr>` for each entry.
+
+```jsx
+function MappedTable(props) {
+  return (
+    <tbody>
```

Comment:
The `<tbody>` could be moved further down to render only the body and the whole `headers` part could be wrapping a `<thead>` element to be rendered on condition - this is more semantically correct.